src/sqlitestorage.cpp

```diff
@@ -1316,13 +1320,19 @@ bool SqliteStorage::Private::saveIncidences(QHash<QString, Incidence::Ptr> &list
 
     for (it = list.constBegin(); it != list.constEnd(); ++it) {
         QString notebookUid = mCalendar->notebook(*it);
-        if ((dbop == DBInsert || dbop == DBUpdate)
-            && !mStorage->isValidNotebook(notebookUid)) {
-            qCWarning(lcMkcal) << "invalid notebook - not saving incidence" << (*it)->uid();
-            continue;
-        } else {
-            (*savedIncidences) << *it;
+        if (dbop == DBInsert || dbop == DBUpdate) {
+            const Notebook &notebook = mStorage->notebook(notebookUid);
+            // Nota here : allow to save/delete incidences in a read-only
```

Comment:
Nit: "nota" sound like a typo.

Reply:
It was a contraction of latine "nota bene" ;) Maybe a bit over contracted actually. I've updated for a more English "notice:" construction.